config/config.exs

```diff
@@ -5,18 +5,17 @@
 # is restricted to this project.
 use Mix.Config
 
+# General application configuration
+config :constable,
+  ecto_repos: [Constable.Repo]
+
 # Configures the endpoint
 config :constable, ConstableWeb.Endpoint,
   url: [host: "localhost"],
   secret_key_base: "tJ+MdrPlKWMpmz7JyJgSu/11xvwnNZo7Sz8IAacy9MM6di3GqackE9iNjhkHI9p8",
-  debug_errors: false,
```

Comment:
This is a default now

Reply:
Is this a `default: true` in `config/dev.exs` or something like that?